pkgs/development/libraries/science/math/tiny-cuda-nn/default.nix

```
@@ -0,0 +1,152 @@
+{ cmake
+, fetchFromGitHub
+, lib
+, ninja
+, pythonPackages
```

```suggestion
, pythonPackages ? null
```
I'm getting the following error from nixpkgs-review:
```
error: Function called without required argument "pythonPackages" at /home/sam.ainsworth/.cache/nixpkgs-review/pr-218166-1/nixpkgs/pkgs/development/libraries/science/math/tiny-cuda-nn/default.nix:5, did you mean "python2Packages", "python3Packages" or "python27Packages"?
(use '--show-trace' to show detailed location information)
nix --experimental-features nix-command --system x86_64-linux eval --json --impure --no-allow-import-from-derivation --expr (import /nix/store/nk3pzcw19gvpyjy2zqcirliczj4sly79-nixpkgs-review-2.8.0/lib/python3.10/site-packages/nixpkgs_review/nix/evalAttrs.nix { allowAliases = false; attr-json = /tmp/tmpzjdn5b5v; }) failed to run, /tmp/tmpzjdn5b5v was stored inspection
https://github.com/NixOS/nixpkgs/pull/218166 failed to build
```
I'm guessing that the easiest solution is to just make this argument optional.

---

Fixed in the new commit